citi-onboarding/Recorda-one-page
# Feature/news section

### **What I did:**
- Created the News component
- Created the news page
- Created the NewsSection
- Deleted the "home" page and its imports
- Added some colors to "theme.ts"

### **How to test:**
- git checkout main
- git pull
- git checkout feature/news_section
- open the "client" directory at the terminal
- write "npm run start" at the terminal
- check if the news section is working
- review the code
